manifests/03_credentials_request_gcp.yaml

```diff
@@ -18,11 +18,14 @@ spec:
     apiVersion: cloudcredential.openshift.io/v1
     kind: GCPProviderSpec
     predefinedRoles:
-      # FIXME: find a replacement for instanceAdmin, since the CSI driver
-      # only needs "compute.instances.[get|attachDisk|DetachDisk]"
-      - roles/compute.instanceAdmin
-      - roles/compute.storageAdmin
-      - roles/iam.serviceAccountUser
+      - "roles/editor"
```

Comment:
And please add a link with source of the permissions, so we can update them when upstream changes.

Reply:
Adding a link/comment to docs. Although it does not mention `compute.disks.get`.

Reply:
Yeah, so please remove it (it's part of compute.storageAdmin anyway) to keep it in sync with the upstream docs.

Reply:
Removing.